File: palindromeNumber/paNu.py
class Solution:
    def isPalindrome(self, x: 'int') -> 'bool':
        if x<0:
            return False
        re = 0
        num = x
        while(num>0):
            re = re*10 + num%10
            num = int(num/10)
        if re==x:
            return True
        else:
            return False

File: palindromeNumber/paNu.java
class Solution {
    public boolean isPalindrome(int x) {
        if(x<0){
            return false;
        }
        int num = x;
        int result = 0;
        while(num !=0){
            result = result*10 + num%10;
            num = num/10;
        }
        if(result == x){
            return true;
        }
        else{
            return false;
        }
    }
}